app/scripts/modules/core/authentication/userMenu/userMenu.directive.html

```diff
@@ -0,0 +1,3 @@
+<li ng-show="authEnabled" class="user-menu">
```

Comment:
We prefer `ng-if` to `ng-show`, unless we really want the internal DOM to be rendered and hidden via CSS (for example, we want some of the wizard pages to be rendered in order to initialize their controllers, but that is a whole other huge mess). That's not to say we _haven't_ littered the codebase with `ng-show`s that should be `ng-if`s, just that we haven't cleaned them up and are trying to avoid them in new code.

Anyway, short story is: this should probably be an `ng-if` unless it needs to be an `ng-show`, which I don't think is the case.